rpc/internal/health_service.go

```diff
@@ -0,0 +1,38 @@
+package internal
+
+import (
+	"context"
+
+	"github.com/evergreen-ci/cedar"
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+)
+
+type healthService struct {
+	services map[string]HealthCheckResponse_ServingStatus
+	env      cedar.Environment
+}
+
+// AttachHealthService attaches the health check service to the given gRPC
+// server.
+func AttachHealthService(env cedar.Environment, services map[string]HealthCheckResponse_ServingStatus, s *grpc.Server) {
+	srv := &healthService{
+		services: services,
+		env:      env,
```

Comment:
Is `env` necessary for this? Seems like it's unused.

Should this return unhealthy when the env context is done and Cedar is shutting down?

Reply:
You are right, eventually we may want to ping the DB or something but for now we can leave it out.